### PR TITLE
automation: Remove ChangeSlaveType steps from resharding workflow and the task itself.

### PR DESCRIPTION
**Pull request by michael-berlin**
@enisoc @aaijazi @alainjobart

### CODE REVIEW

File: go/cmd/vtworker/vtworker.go [hunk 1]
```diff
@@ -78,9 +78,11 @@ func main() {
 		go func() {
 			if err := wi.WaitForCommand(worker, done); err != nil {
 				log.Error(err)
+				log.Flush()
```
Comment (alainjobart) [inline go/cmd/vtworker/vtworker.go]: Not sure that will import properly, I know google's internal log doesn't have that... logutil has something I think.

**Comment by alainjobart** [comment]
LGTM after comment addressed.

File: go/cmd/vtworker/vtworker.go (hunk 1 above)
Comment (enisoc) [inline go/cmd/vtworker/vtworker.go]: It should be safe to replace log.Flush() with logutil.Flush(). That package wraps all the differences.

File: go/vt/worker/topo_utils.go [hunk 2]
```diff
@@ -18,24 +18,44 @@ import (
...
 func FindHealthyRdonlyEndPoint(ctx context.Context, wr *wrangler.Wrangler, cell, keyspace, shard string) (topo.TabletAlias, error) {
-	endPoints, _, err := wr.TopoServer().GetEndPoints(ctx, cell, keyspace, shard, topo.TYPE_RDONLY)
-	if err != nil {
-		return topo.TabletAlias{}, fmt.Errorf("GetEndPoints(%v,%v,%v,rdonly) failed: %v", cell, keyspace, shard, err)
-	}
-	healthyEndpoints := make([]topo.EndPoint, 0, len(endPoints.Entries))
-	for _, entry := range endPoints.Entries {
-		if len(entry.Health) == 0 {
-			healthyEndpoints = append(healthyEndpoints, entry)
+	waitDeadline := time.Now().Add(*waitForHealthyEndPointsTimeout)
```
Comment (enisoc) [inline go/vt/worker/topo_utils.go]: Would be nice to do this with a context.WithTimeout(ctx, ...).

Comment (michael-berlin) [inline go/vt/worker/topo_utils.go]: DONE

**Comment by michael-berlin** [comment]
@enisoc PTAL I've implemented it as you recommended and would merge it after that.

**Comment by enisoc** [comment]
LGTM

**Pull request merged**